### PR TITLE
Add styled-components dependencies

### PR DESCRIPTION
**Pull request by svrin**
Allows to process styled-components with stylelint.

Basicly styled-components is a way to allow for styling react classes in react. For example:

```
  const RedHeadline = styled.h1`
    color: red
  `,
```

And stylelint can lint them via these 2 dependencies.

Example .stylelintrc:

```
{
  "processors": ["stylelint-processor-styled-components"],
  "extends": [
    "stylelint-config-recommended",
    "stylelint-config-styled-components"
  ]
}
```

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -24,3 +24,5 @@ npm install -g stylelint-config-prettier@5.2.0
 npm install -g stylelint-config-css-modules@1.4.0
 npm install -g prettier@1.18.2
 npm install -g stylelint-prettier@1.1.1
+npm install -g stylelint-config-styled-components@0.1.1
+npm install -g stylelint-processor-styled-components@1.5.2
```
Comment (lolgab) [inline scripts/install.sh]: Why did you choose this version instead of the latest 1.8.0 ?

Comment (svrin) [reply scripts/install.sh]: Its the latest one we use in our projects and I know that it works with 1.5.2. If its easier for you I can test it with 1.8.0 and change the PR accordingly.

Comment (lolgab) [reply scripts/install.sh]: Yes, it could be cool :)

**Comment by svrin** [comment]
Thx!

**Pull request merged**